shells/lib/env/env-base.js

```diff
@@ -1,4 +1,5 @@
 import {Env} from '../arcs.js';
+import {Runtime} from '../../../build/runtime/runtime.js';
```

Comment:
Why is env-base being changed at all?

Reply:
IMO, `env` and `Runtime` are solutions to the same problem. We shouldn't try to use them both.

Reply:
Sorry, Scott. I should have put more into the initial comment.

This entire PR a hack to unblock Maria, while I work on the correct solution. Shane asked me to look at a quick way, as well as a long-term way, to help her deal with the fact that she needs to instantiate a plan without an arc. Instantiation needs a loader and a context, both of which are currently pretty much singletons, so this PR is for pushing the loader directly into the runtime code, without attaching it to an arc. The plan for today is to do the same for the context.